web/libs/datamanager/src/components/MainView/GridView/ImagePreview.tsx

```diff
@@ -81,9 +86,8 @@ const ImagePreview = observer(({ task, field }: ImagePreviewProps) => {
       const scaledWidth = img.naturalWidth * imageScale;
       const scaledHeight = img.naturalHeight * imageScale;
       // how much should we zoom image in to cover container
-      const coverScale = Math.max(containerRect.width / scaledWidth, containerRect.height / scaledHeight);
+      // const coverScale = Math.max(containerRect.width / scaledWidth, containerRect.height / scaledHeight);
```

Comment:
are you looking to put this back at some point? or should this be removed?
```suggestion
```

Reply:
Yeah, this might be helpful, so I'll leave it here for now